dev/start.sh

```diff
@@ -60,6 +60,10 @@ export ZOEKT_HOST=localhost:3070
 export USE_ENHANCED_LANGUAGE_DETECTION=${USE_ENHANCED_LANGUAGE_DETECTION:-1}
 export GRAFANA_SERVER_URL=http://localhost:3370
 
+# Caddy / HTTPS configuration
+export SOURCEGRAPH_HTTPS_DOMAIN=sourcegraph.test
+export SOURCEGRAPH_HTTPS_PORT=3443
```

Comment:
Sorry if I wasn't clear, this is what I had in mind so it is overridable
```suggestion
export SOURCEGRAPH_HTTPS_DOMAIN=${$SOURCEGRAPH_HTTPS_DOMAIN:-sourcegraph.test}
export SOURCEGRAPH_HTTPS_PORT=${$SOURCEGRAPH_HTTPS_PORT:-3443}
```